test/hotspot/jtreg/testlibrary_tests/ir_framework/tests/TestBadFormat.java

```diff
@@ -843,12 +843,330 @@ public void mustSpecifyAtLeastOneConstraint2() {
     public void mustSpecifyAtLeastOneConstraint3() {
     }
 
+
+    @FailCount(3)
+    @Test
+    @IR(failOn = IRNode.CALL,
+        applyIfCPUFeature = {"sve", "true"},
+        applyIfCPUFeatureAnd = {"avx", "true", "sve", "true"})
+    @IR(failOn = IRNode.CALL,
+        applyIfCPUFeatureAnd = {"sve", "true", "avx", "true"},
+        applyIfCPUFeatureOr = {"sve", "true", "avx", "true"})
+    @IR(failOn = IRNode.CALL,
+        applyIfCPUFeature = {"sve", "true"},
+        applyIfCPUFeatureAnd = {"sve", "true", "avx", "true"},
+        applyIfCPUFeatureOr = {"sve", "true", "avx", "true"})
+    public void onlyOneApplyIfCPUFeature() {}
+
+    @FailCount(3)
```

Comment:
Why are there 3 failures?

Reply:
Generally, the count is not always as I would expect. Can we fix that? You should at least comment what fails, and why.

I would expect there to be a single failure per rule. But that seems not to be the case.